server/models/User.js

```diff
@@ -43,73 +46,72 @@ const userSchema = mongoose.Schema({
         ref: 'Club'
     }],
     */
-    token: {
-        type: String
-    },
-    tokenExp: {
-        type: Number
-    }
-})
+  token: {
+    type: String,
+  },
+  tokenExp: {
+    type: Number,
+  },
+});
 
 //유저 모델에 유저 정보를 저장하기 전에 아래 callback ftn이 실행됨
-userSchema.pre('save',function(next){
-    var user = this;
-    //비밀번호가 변경된 직후 한번만 암호화 진행
-    if(user.isModified('password')){
-        //비밀번호 암호화
-        bcrypt.genSalt(saltRounds,function(err,salt){
-            if(err) return next(err);
-            bcrypt.hash(user.password,salt,function(err,hash){
-                if(err) return next(err);
-                user.password = hash;
-                next();
-            })
-        })
-    } else {
+userSchema.pre("save", function (next) {
+  var user = this;
+  //비밀번호가 변경된 직후 한번만 암호화 진행
+  if (user.isModified("password")) {
+    //비밀번호 암호화
+    bcrypt.genSalt(saltRounds, function (err, salt) {
+      if (err) return next(err);
+      bcrypt.hash(user.password, salt, function (err, hash) {
+        if (err) return next(err);
+        user.password = hash;
         next();
-    }
-})
+      });
+    });
+  } else {
+    next();
+  }
+});
 
 //비밀번호를 확인하는 comparePassword 메소드 생성
-userSchema.methods.comparePassword = function(plainPassword, cb){
-    //bcrypt 이용하여 비밀번호 확인
-    bcrypt.compare(plainPassword, this.password, function(err, isMatch){
-        if(err) return cb(err)
-        cb(null, isMatch)
-    })
-}
+userSchema.methods.comparePassword = function (plainPassword, cb) {
+  //bcrypt 이용하여 비밀번호 확인
+  bcrypt.compare(plainPassword, this.password, function (err, isMatch) {
+    if (err) return cb(err);
+    cb(null, isMatch);
+  });
+};
 
 //토큰을 생성하는 generateToken 메소드 생성
-userSchema.methods.generateToken = function(cb) {
-    var user = this;
-    //jwt.sign 의 첫 인자는 payload, 두번째 인자는 비밀키 값
-    //유저 아이디로 'secretToken'이라는 토큰 생성
-    var token = jwt.sign(user._id.toHexString(),config.tokenKey);
-    var oneHour = moment().add(1, 'hour').valueOf();
+userSchema.methods.generateToken = function (cb) {
+  var user = this;
+  //jwt.sign 의 첫 인자는 payload, 두번째 인자는 비밀키 값
+  //유저 아이디로 'secretToken'이라는 토큰 생성
+  var token = jwt.sign(user._id.toHexString(), config.tokenKey);
+  var oneHour = moment().add(1, "hour").valueOf();
 
-    user.tokenExp = oneHour;
-    user.token = token;
-    user.save(function(err,user){
-        if(err) return cb(err);
-        cb(null,user);
-    })
-}
+  user.tokenExp = oneHour;
+  user.token = token;
+  user.save(function (err, user) {
```

Comment:
```suggestion
  user.save(function(err, user) {
```

요기는 space 붙이는 게 맞는 것 같아용